src/util/LOG.cxx

```diff
@@ -1,19 +1,27 @@
 #include "LOG.hxx"
 
+#ifdef __ANDROID__
+#include <android/log.h>
+#endif
+
 Mutex LOG::StreamMutex;
 
 LOG::LOG(LogType type) : m_LogType(type) {}
 
 LOG::~LOG()
 {
   LockGuard lock(StreamMutex);
-  string Message = getTimeStamp() + LOG_PREFIX[m_LogType] + m_Logger.str();
+  string message = getTimeStamp() + LOG_PREFIX[m_LogType] + m_Logger.str();
   if (!getenv("TERM"))
   {
 #ifndef DEBUG
     if (m_LogType != LOG_DEBUG)
 #endif
-      std::cout << Message << std::endl;
+#ifdef __ANDROID__
+      __android_log_print(ANDROID_LOG_INFO, "Cytopia", "%s", Message.c_str());
```

Comment:
Are you sure it is compiling now?

Message -> message?

Reply:
ah.. i used "refactor - rename" in visual studio and it skipped the ifdef. fixed now. since we don't have an android build yet, it did not fail